content/docs/ref/runner.md

```diff
@@ -36,8 +36,8 @@ Any [generic option](/doc/ref) in addition to:
   *current price*].
 - `--cloud-region={us-west,us-east,eu-west,eu-north,...}`:
   [Region](https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#cloud-regions)
-  where the instance is deployed. Also accepts native cloud regions [default:
-  `us-west`].
+  where the instance is deployed. Also accepts native AWS/Azure region or GCP
+  zone [default: `us-west`].
```

Comment:
```suggestion
  where the instance is deployed. Also accepts native AWS/Azure regions or GCP
  zones [default: `us-west`].
```

Plural, perhaps? 🤔

Reply:
Can you specify multiple in one go?

Reply:
No

Reply:
Still, plural feels more natural to my mind. Perhaps just a non–native brain artifact.

Reply:
both are grammaticisionally correk. But singular is more precise for this `--flag`.